pkg/testhelper/testhelper.go

```diff
@@ -157,6 +158,8 @@ const (
 	ListeningPortType            = "Listening Port"
 	DeclaredPortType             = "Declared Port"
 	ContainerPort                = "Container Port"
+	HostPort                     = "Host Port"
```

Comment:
It looks like object types are used to define the type of a report object and are only used as parameters to the `SetType()` function whereas field types add new data to a report object and are used as parameters to the `AddField()` function. See how `DeclaredPortType` is used for reference.

Reply:
Can you re-review to see if I'm doing it correct?

Reply:
Yes. That's the way I think those types are meant to be used.